src/FileHelpers.jl

```diff
@@ -1,5 +1,11 @@
 module FileHelpers
-import Pluto: is_pluto_notebook
+import Pluto: Pluto, is_pluto_notebook, without_pluto_file_extension
+using Base64
+using FromFile
+
+
+@from "Export.jl" import Export: Export, generate_html
+pluto_version = Export.try_get_exact_pluto_version()
```

Comment:
this should not be a toplevel definition because it will get hardcoded into the precompilation cache